cloud-operations/scheduled-asset-inventory-export-bq/cf/main.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Cloud Function module to export data for a given day.
+
+This module is designed to be plugged in a Cloud Function, attached to Cloud
+Scheduler trigger to create a Cloud Asset Inventory Export to BigQuery.
+
+'''
+
+import base64
+import datetime
+import json
+import logging
+import os
+import warnings
+
+import click
+
+from google.api_core.exceptions import GoogleAPIError
+from google.cloud import asset_v1
+
+import googleapiclient.discovery
+import googleapiclient.errors
+
+def _configure_logging(verbose=True):
+  """Basic logging configuration.
+  Args:
+    verbose: enable verbose logging
+  """
+  level = logging.DEBUG if verbose else logging.INFO
+  logging.basicConfig(level=level)
+  warnings.filterwarnings('ignore', r'.*end user credentials.*', UserWarning)
+
+@click.command()
+@click.option('--organization', required=True,
+              help='Organization ID')
+@click.option('--bq-project', required=True,
+              help='Bigquery project to use.')
+@click.option('--bq-dataset', required=True,
+              help='Bigquery dataset to use.')
+@click.option('--bq-table', required=True,
+              help='Bigquery table name to use.')
+@click.option('--read-time', required=True,
+              help='Day to take an asset snapshot in the format \'YYYYMMDD\'. \
+              If not specified run for the current day. Export will run for \
+              the midnight of the specified day.')
+@click.option('--verbose', is_flag=True, help='Verbose output')
+def main_cli(organization=None, bq_project=None, bq_dataset=None, bq_table=None, read_time=None, verbose=False):
+  """Trigger Cloud Asset inventory export to Bigquery. Data will
```

Comment:
Can you use either single or double quotes consistently? (I prefer single but YMMV).